go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/dgryski/go-jump v0.0.0-20170409065014-e1f439676b57
 	github.com/google/go-jsonnet v0.14.0
 	github.com/jsonnet-bundler/jsonnet-bundler v0.1.1-0.20190930114713-10e24cb86976
+	github.com/oklog/run v1.1.0
```

Comment:
Why not use `errgroup` instead? It's a part of the go stdlib

Reply:
To be fair, I'm quite new to developing in Go so I just followed the recommendation made on #1169 and how it is done on Thanos.
IMHO, it makes sense to avoid using an external dependency when there is an alternative in the stdlib. On the other hand, I have no idea what the (technical) differences are between oklog's run and the stdlib's errgroup.

@brancz Any reason to use oklog/run over errgroup?

Reply:
errgroup is for goroutines of the same "type", rungroups are for orchestrating goroutines that have distinct purposes